openmdao/components/eq_constraint_comp.py

```diff
@@ -66,8 +76,24 @@ def __init__(self, name=None, eq_units=None, lhs_name=None, rhs_name=None, rhs_v
         mult_val : int, float, or np.array
             Default value for the LHS multiplier of the given output.  Must be compatible
             with the shape (optionally) given by the val or shape option in kwargs.
+        normalize : bool
+            Specifies whether or not the resulting output should be normalized by a quadratic
```

Comment:
Its not really quadratic. Its effectively linear almost everywhere, except around 0.